src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,118 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import core.basesyntax.service.UserHasNoLoginException;
+import core.basesyntax.service.UserNullException;
+import core.basesyntax.service.UserPasswordNullException;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
 public class HelloWorldTest {
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    static void initial() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+     void createValidUser() {
+        user = new User();
+        user.setAge(42);
+        user.setLogin("labudabudapta");
+        user.setPassword("motorama");
+    }
+
+    @Test
+    void userIsNull_NotOk() {
+        User userNull = null;
+        assertThrows(UserNullException.class,
+                () -> registrationService.register(userNull),
+                "user is null\n");
+    }
+
+    @Test
+    void userIsYoungerThan18_NotOk() {
+        user.setAge(12);
+        assertThrows(RuntimeException.class,
+                () -> registrationService.register(user),
+                "condition break: user with age up to 18 was added\n");
+    }
+
+    @Test
+    void userAgeIsNegative_NotOk() {
+        user.setAge(-25);
+        assertThrows(RuntimeException.class,
+                () -> registrationService.register(user),
+                "condition break: user's age can't be negative\n");
+    }
+
+    @Test
+    void userLoginLesserThanSixCharacters_NotOk() {
+        user.setLogin("Robin");
+        assertThrows(RuntimeException.class,
+                () -> registrationService.register(user),
+                "too short login\n");
+    }
+
+    @Test
+    void userLoginIsNull_NotOk() {
```

Comment:
void register_nullLogin_notOk()

you have to change naming of your Tests. 
In description to this task we have an example, how test has to be called. For example:
"register_nullAge_notOk" after underscope you have to write the next word starts from small letter.